packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeComponent.tsx

```diff
@@ -84,15 +86,37 @@ export const ModelsTreeComponent = (props: ModelTreeProps) => {
     viewport?.invalidateScene();
   }, [viewport, availableModels]);
 
-  const hideAll = useCallback(() => {
+  const hideAll = useCallback(async () => {
     viewport?.changeModelDisplay(availableModels, false);
+    viewport?.clearAlwaysDrawn();
+    if (iModel) {
+      await toggleAllCategories(
+        IModelApp.viewManager,
+        iModel,
+        false,
+        undefined,
```

Comment:
I think `viewport` should be passes to this function. And probably we do not want to toggle categories in all viewports.

Reply:
fixed